M2/Macaulay2/d/basic.d

```diff
@@ -18,7 +18,8 @@ export hash(e:Expr):hash_t := (
      is MysqlConnectionWrapper do hash_t(237489) -- improve this later!
      is MysqlFieldWrapper do hash_t(23748) -- improve this later!
      is MysqlResultWrapper do hash_t(2374) -- improve this later!
-     is CodeClosure do hash_t(73889)       -- improve this later!
+     is PseudocodeClosure do hash_t(73889)       -- improve this later!
```

Comment:
Maybe align the "-- improve this later!" comments? (unless you want to just remove them)

Reply:
yeah I'm not sure what these comments are meant for